### ISSUE TITLE
Deploy new King County Parks dashboard

### ISSUE THREAD

**Issue by jblehr** [OutdoorRD/trails-viz#193]
See original issue here:
https://github.com/OutdoorRD/king-county-parks/issues/23#issuecomment-2101522495

Likely following the docs here:
https://github.com/OutdoorRD/trails-viz/wiki/Adding-New-Project

**Comment by jblehr** [comment]
I am running into some errors with Travis. I sent an email to their support email. Here's the summary of the issue:

First, I am seeing the following error on our Travis account page:
“We are unable to start your build at this time. You exceeded the number of users allowed for your plan. Please review your plan details and follow the steps to resolution.”
 
When I go to the “Requests” tab of the “trails-viz” repo here:
https://app.travis-ci.com/github/OutdoorRD/trails-viz/requests?serverType=git, I see the following error: “Owner OutdoorRD does not have enough credits.”
 
When I go to the plan (https://app.travis-ci.com/account/plan) page, I see a different error:
“Builds have been temporarily disabled for private and public repositories due to a negative credit balance. Please go to the Plan page to replenish your credit balance.”

However, I don’t see a way to replenish credits.

**Comment by jblehr** [comment]
Noting here that Travis replenished our credits under an OSS license.

And the v1 dashboard was deployed, after solving another issue raised in slack [here](https://recsetgo.slack.com/archives/CMV23TDRB/p1715799165184589).

This issue is still open until the team checks the dashboard for accuracy.

**Comment by jblehr** [comment]
Closing this issue after #196 was merged and we don't have any outstanding issues on the data.
